webapp/components/test/wpt-interop.html

```diff
@@ -73,6 +67,35 @@
           .find(c => c.args.length && c.args[0].pathname === '/api/interop');
       });
     });
+
+    suite('interopScore()', () => {
+      const allFail = [10];
+      const allPass = [10];
+      for (let n = 1; n < 10; n++) {
```

Comment:
`n` should be initialized to `2` since the first iteration describes a state with 2 products (e.g. `[10, 0]` and `[0, 10]`). You may want to add a test for the "1 product" case, but I don't know how likely we are to ever need that (or how interested people will be in the "interoperability" of a single browser).

Reply:
Good point.